### ISSUE TITLE
`<`/`>` should not be replaced inside `\text{}`

### ISSUE THREAD

**Issue by dummy-index** [jmnote/SimpleMathJax#58]
Premise 1: In the current `<math>` tag, character references are not dereferenced and are passed to MathJax as is (Html::element() encodes `&` as `&amp;` in the HTML message, the browser decodes `&amp;` back to `&` for display, and MathJax reads the displayed string rather than the HTML source).
Premise 2: This specification is roughly the same as Extension:Math. This specification is different from math mode using delimiters or environments (these send character references directly to the browser, which decodes the character references).
Premise 3: For the time being, we'll leave it as is, with specifications close to Extension:Math.

Replacing `<` and `>` with `\lt ` and `\gt ` seems unnecessary if you assume using Html:element(). All strings sandwiched between `<math>` and `</math>` are escaped by Html:element(), so even if `<b>` is written inside, it seems to pass to MathJax as a string, not an HTML fragment.

And the obvious problems:
- Replacing all inequality signs obviously has a negative impact on mhchem's arrow notation, so we didn't replace it with `<chem>`, but considering the possibility of using `\ce{}` in `<math>`, I would like to stop it completely.
- In `\text{}`, the lt/gt macros do not work, but SimpleMathJax replaces it uniformly, so if you write `<`, it will be displayed as `\lt `. `\unicode{x3c}` doesn't work in `\text{}`...(ok, `\require{textmacros}`)
- By the way, `\>` is replaced with `\;`, but shouldn't it be the same width as `\:`?

I also want to use Sanitizer::validateAttributes() for inheriting attributes.

https://github.com/users/dummy-index/projects/1/views/1?pane=issue&itemId=148204347

**Comment by dummy-index** [comment]
In addition, the href and style attributes output by the MathJax HTML extension should also be sanitized. However, if you try to do this with a postFilter, you would need a JavaScript implementation of a sanitizer. For now, how about enabling the [safe extension](https://docs.mathjax.org/en/v3.2/options/safe.html)?

**Comment by dummy-index** [comment]
Even without explicitly loading mhchem, autoload works, so logically `\ce{}` can be used without any issues. However, from a UX perspective, loading it at the config stage has the advantage of allowing parallel loading, which enables faster rendering. This means that by using encounters with `<math chem>` or `<chem>` during parsing as hints for parallel loading, it is possible to avoid loading mhchem.js unnecessarily on pages that do not use chem. (On pages that use delimiters, preloading will no longer occur, and rendering may be slower, but it would just be under the same conditions as other extensions.)
- (Isn't it too late that the loading of tex-chtml.js starts after `wikipage.content` anyway? ...I want to carefully verify this, so I'll create a separate issue.)
